timeless/templates/auth/forgot_password.html

```diff
@@ -15,4 +15,4 @@ <h1>Forgot your password?</h1>
     <input type="submit" value="Forgot my password">
   </form>
 {% endblock %}
-{% endcache %}
\ No newline at end of file
+{% endcache %}
```

Comment:
@marceloamadeu Do we need to cache forgot password page?

Reply:
@ammaratef45 Truth. I also believe you do not have to. Retired.